### ISSUE TITLE
XLS Support

### ISSUE THREAD

**Issue by mishu-** [h2non/filetype.py#66]
Is it doable to also check for XLS?

**Comment by h2non** [comment]
Yes, it is. It was already implemented in the Go port of this package:
https://github.com/h2non/filetype#documents

PR would be very welcome!

**Comment by Agent-Hellboy** [comment]
Hi @h2non 
The magic number of all the office documents are same how will I distinguish between doc, xls, and ppt file.
I have also checked the ```https://github.com/file/file``` but that was tough for me to understand.``  

``
Word Document | .doc | D0 CF 11 E0 A1 B1 1A E1
Excel Document | .xls | D0 CF 11 E0 A1 B1 1A E1
PowerPoint Document | .ppt | D0 CF 11 E0 A1 B1 1A E1
Visio Document | .vsd | D0 CF 11 E0 A1 B1 1A E1
``
